cmd/abci-cli/abci-cli.go

```diff
@@ -228,9 +205,10 @@ func cmdEcho(c *cli.Context) error {
 	if len(args) != 1 {
 		return errors.New("Command echo takes 1 argument")
 	}
-	res := client.EchoSync(args[0])
-	rsp := newResponse(res, string(res.Data), false)
-	printResponse(c, rsp)
+	resEcho := client.EchoSync(args[0])
+	printResponse(c, response{
+		Data: resEcho.Data,
```

Comment:
resInfo.Data is a string.  Arguably resInfo.Data should be a []bytes or be renamed something to be consistent, but maybe someone should create a new pull request with proposed changes.  I'm not sure what's best.